utils/multisigexecutor.js

```diff
@@ -32,7 +33,7 @@ class MultisigExecutor {
       signature = ethers.utils.joinSignature(split);
 
       // Call "execute" on the Multisig wallet with data and signatures
-      const executeTransaction = await this._multisigWrapper.contract.execute(contractAddress, 0, data, signature, { gasLimit: 2000000 });
+      const executeTransaction = await this._multisigWrapper.contract.execute(contractAddress, 0, data, signature, this._overrides);
```

Comment:
There is a multisig transaction that failed (silently) on CI: https://app.circleci.com/pipelines/github/argentlabs/argent-contracts/451/workflows/5f21b4a6-2bfe-491f-8113-527c50ddf6f0/jobs/485

I wonder if this could be because you have reduced the gasLimit from 2M to 300k here.

Reply:
Hmm, I did not see any fail transactions when running the deployment scripts, but I guess we set it to 2M for a reason. I will fix it.